src/Cake.Common/Tools/DotNetCore/Execute/DotNetCoreExecutor.cs

```diff
@@ -40,7 +40,6 @@ public void Execute(FilePath assemblyPath, string arguments, DotNetCoreSettings
             {
                 throw new ArgumentNullException("assemblyPath");
             }
-
```

Comment:
@gep13: I usually don't have a new line before guard statements like that and I think it's like there in most of the code base. We do have a rule that says that there can be no empty lines before a closing brace like this:

```csharp
if(lol)
{
  DoMagic();

}
```